docs/user/modules/python/pages/pyfeelppmor/reducedbasis.adoc

```diff
@@ -4,112 +4,355 @@ include::user:ROOT:partial$header-macros.adoc[]
 
 == Import
 
-NOTE: The work has not yet been merged in `develop`. To include the module at current state, you need to checkout the branch https://github.com/feelpp/feelpp/tree/feature/pyrb[`feature/pyrb`], and move to the directory https://github.com/feelpp/feelpp/tree/feature/pyrb/mor/pyfeelpp-mor/feelpp/mor[mor/pyfeelpp-mor/feelpp/mor].
+NOTE: The work has not yet been merged in `develop`. To include the module at current state, you need to checkout the branch https://github.com/feelpp/feelpp/tree/feature/pyrb[`feature/pyrb`].
 
 Line to import the module :
 
 [source, python]
 ----
-from reducedbasis.reducedbasis import *
+import feelpp.mor.reducedbasis.reducedbasis_timeOffline as mor_rb
 ----
 
+NOTE: This import load the module for offline generation, associated to the time-dependent case. For the time-independent case, you can import `feelpp.mor.reducedbasis.reducedbasisOffline` instead.
+
 To set the environment, those module also need to be imported :
 
 [source, python]
 ----
-import sys
+import sys, os
 from feelpp.toolboxes.heat import *
-from feelpp.toolboxes.core import *
-from feelpp.mor import *
+import feelpp.toolboxes.core as core
+from feelpp.operators import mass
+import feelpp.mor as mor
 import feelpp
 ----
 
 
+== Set the environment
+
+[source, python]
+----
+config = feelpp.globalRepository("reducedbasis")
+sys.argv = ["reducedbasis"]
+o = core.toolboxes_options("heat")
+o.add(mor.makeToolboxMorOptions())
+
+e = feelpp.Environment(sys.argv, opts=o, config=config)
+----
+
+
 == Set the model
 
-To declare the objects, use these lines :
+Set the parameters nedded to the model :
+- `config_file` : path to the cfg file
+- `dim` : the dimension of the case
+- `order` : the order of discretization (`1` for stem:[\mathbb{P}_1], `2` for stem:[\mathbb{P}_2])
+- `time_dependant` : is the case stationnary or transient ? (for now always to `False`)
+- `name` : name of the CRB case
 
 [source,python]
 ----
-DIM = 2
-heatBox=heat(dim=DIM,order=1)
-heatBox.init()
-model = toolboxmor(dim=DIM, time_dependent=False)
-model.setFunctionSpaces( Vh=heatBox.spaceTemperature())
+config_file = "thermal-fin.cfg"
```

Comment:
this is not robust. Do you create the file ? or should it be available somewhere ?